### PR TITLE
Inactivity updates test generation, implement new participation flag updates testing

### PR DESCRIPTION
**Pull request by protolambda**
- Update `epoch_processing` test generator to output tests for missing tests
- Implement new `process_participation_flag_updates` tests

### CODE REVIEW

File: tests/formats/epoch_processing/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The provided pre-state is already transitioned to just before the specific sub-t
 Sub-transitions:
 
 - `justification_and_finalization`
-- `inactivity_penalty_updates`
+- `inactivity_penalty_updates` (Altair)
```
Comment (hwwhww) [inline tests/formats/epoch_processing/README.md:36]: It seems a minor leftover from #2395

```suggestion
- `inactivity_updates` (Altair)
```

File: tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_participation_flag_updates.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+@with_altair_and_later
+@spec_state_test
+def test_random_genesis(spec, state):
+    random_flags(spec, state, 11)
+    yield from run_process_participation_flag_updates(spec, state)
+
+
+@with_altair_and_later
+@spec_state_test
+def test_zeroing(spec, state):
```
Comment (hwwhww) [inline tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_participation_flag_updates.py:73]: How about
```suggestion
def test_current_epoch_zeroed(spec, state):
```

File: tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_participation_flag_updates.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    next_epoch_via_block(spec, state)
+    random_flags(spec, state, 12, current=False)
+    state.current_epoch_participation = [0] * len(state.validators)
+    yield from run_process_participation_flag_updates(spec, state)
+
+
+@with_altair_and_later
+@spec_state_test
+def test_prev_zeroed(spec, state):
```
Comment (hwwhww) [inline tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_participation_flag_updates.py:82]: How about 

```suggestion
def test_previous_epoch_zeroed(spec, state):
```

File: tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_participation_flag_updates.py [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+def run_process_participation_flag_updates(spec, state):
+    old = state.current_epoch_participation
+    yield from run_epoch_processing_with(spec, state, 'process_participation_flag_updates')
+    assert state.current_epoch_participation == [0] * len(state.validators)
+    assert state.previous_epoch_participation == old
+
+
+@with_altair_and_later
+@spec_state_test
+def test_zeroed(spec, state):
```
Comment (hwwhww) [inline tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_participation_flag_updates.py:24]: How about

```suggestion
def test_all_zeroed(spec, state):
```

**Review by hwwhww — APPROVED** [review]
LGTM

**Review by djrtwo — COMMENTED** [review]
I built these with alpha.6, btw

Looks good. two minor comments

**Review by djrtwo — APPROVED** [review]
applied code review. good to go after green build

**Pull request merged**